### DISCUSSION TITLE
Release a beta to the play store

### DISCUSSION THREAD

**Discussion by SeanDylan** [LawnchairLauncher/lawnchair#4212] [Ideas]
Since it's out of its alpha stage it would be cool so that more people can download it

**Comment by arpel** [comment]
Totally concur on this one !

**Reply by arpel** [reply]
It's now published on the PlayStore ! 
Use the link in the announcement from the website to get access to the Beta version.

**Reply by arpel** [reply]
Yet it could be updated "a bit more" (as no update at all until now)
